infra/helper.py

```diff
@@ -416,11 +416,14 @@ def build_image(args):
   return 1
 
 
-def build_fuzzers(args):
+def build_fuzzers(args, skip_build_image=False):
   """Build fuzzers."""
   project_name = args.project_name
-  if not _build_image(args.project_name):
-    return 1
+  if not skip_build_image:
```

Comment:
nit: prefer

```python
if skip_build_image:
  ...
else:
```

Rather than the inverted case to avoid the double negative to make it easier to read.